Library/Formula/sfml.rb

```diff
@@ -27,9 +17,45 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
 
   def install
-    system "cmake", ".", "-DCMAKE_INSTALL_FRAMEWORK_PREFIX=#{frameworks}/", *std_cmake_args
+    args = std_cmake_args
+    args << "."
+
+    args << "-DFREETYPE_LIBRARY=#{Formula["freetype"].opt_lib}/libfreetype.dylib"
+    args << "-DGLEW_LIBRARY=#{Formula["glew"].opt_lib}/libGLEW.dylib"
+    args << "-DGLEW_INCLUDE_PATH=#{Formula["glew"].opt_include}"
+    args << "-DJPEG_LIBRARY=#{Formula["jpeg"].opt_lib}/libjpeg.dylib"
+    args << "-DSNDFILE_LIBRARIES=#{Formula["libsndfile"].opt_lib}/libsndfile.dylib"
+    args << "-DSNDFILE_INCLUDE_DIR=#{Formula["libsndfile"].opt_include}"
```

Comment:
`superenv` means we don't need to be explicit about these things. Could we install the extlibs then just delete them?